modules/apps/commerce/commerce-product-ddm-impl/src/main/java/com/liferay/commerce/product/ddm/internal/DDMHelperImpl.java

```diff
@@ -299,7 +299,7 @@ private String _createDDMFormRuleOutputMapping(DDMForm ddmForm) {
 
 	private DDMForm _getDDMForm(
 		Locale locale, boolean ignoreSKUCombinations, boolean optional,
-		boolean publicStore,
+		boolean publicStore, boolean required,
```

Comment:
@Commando18 This is not good fix.. Parameter optional is one that tells if option is required or optional. You added unnecessary  redundancy here.

Reply:
Ok, thanks, I will use that one then